docs/guides/headless-cms/content-translations.md

```diff
@@ -0,0 +1,155 @@
+
+   - Use `translations` for the key.
+   - For the Languages Collection field, select the `languages` collection you created in Step 1.
+   - For Language Indicator Field, you could use the `code` or `name` fields depending on personal preference.
+   - For Language Direction Field, choose the `direction` field.
+   - You can also select a Default Language by entering the primary key or `code` for that language.
+
+:::tip
+
+By default Directus will name this junction collection `articles_translations` following the convention
+`{collection_name}_translations`. If you want more control over the naming of your translation collection and the fields
+within, you can choose to Continue in Advanced Field Creation Mode.
+
+:::
+
+## Add The Fields To Be Translated
+
+4. Navigate to the newly created `articles_translations` collection and create the fields for the content you'll
+   translate into different languages.
+
+   Here's a sample schema for the collection.
+
+   ```md
+   articles_translations
+
+   - title (Type: String, Interface: Input )
+   - slug (Type: String, Interface: Input)
+   - summary (Type: Text, Inteface: Textarea)
+   - content (Type: Text, Inferface: WYSIWYG)
+   ```
+
+   ![Data Model settings screen for the Articles Translations collection is displayed. The following fields are shown: id, articles_id, languages_code, title, slug, summary, content.](https://marketing.directus.app/assets/e2ee5c61-3449-40cf-9cd4-24f9edb2ae6a.png?key=doc)
+
+## Test By Creating Content
+
+Everything is now setup to allow you to translate Articles into multiple languages so it's time to test what you've
+built.
+
+5. Before you create your first article, you'll need to add a few items to the `languages` collection. Navigate to the
+   Content module > Languages collection and add two languages.
+
+   ![The table layout for the Languages collection is shown. There are two items displayed. One for English and one for French.](https://marketing.directus.app/assets/9afe044e-405c-46e7-851f-2a400f3672ea.png?key=doc)
+
+6. In the Articles collection, create a new article and scroll down to the Translations field and start translating your
+   content.
+
+   You can show one language at a time or show two languages side by side.
+
+   ![A Directus item detail page is shown using the Translations interface. Editable fields for the English and French translations for an article are displayed side-by-side. Fields include Title, Slug, Summary, and Content. ](https://marketing.directus.app/assets/64eee60e-605b-4d2c-809f-83557ed0d5ef.png?key=doc)
+
+   There is also a progress indicator for the status of each language.
+
+   ![A dropdown menu with a progress indicator for English and French languages are shown. The progress is less than half and shown in red.](https://marketing.directus.app/assets/78a895e9-744e-4d33-9d87-3c5ca4c784e2.png?key=doc)
+
```

Comment:
What's missing here is a quick demo of how to interact with this data via the API